scipy/optimize/_highs/src/HConfig.h

```diff
@@ -0,0 +1,4 @@
+#ifndef SCIPY_HIGHS_HCONFIG_H
+#define SCIPY_HIGHS_HCONFIG_H
+
```

Comment:
A comment that right now this file does nothing and `SCIPY_HIGHS_CONFIG_H` isn't used elsewhere would be useful to prevent confusion I think.

Reply:
Missed this comment and didn't get it in the last round of updates.  Going to make sure CI comes back green and then I'll push your suggested comment